code/src/components/app-accordion/app-accorion-container/AppAccordionContainer.tsx

```diff
@@ -1,6 +1,6 @@
 import Accordion from "@mui/material/Accordion";
 import { AppAccordionProps } from "@/components/app-accordion/app-accorion-container/AppAccordionContainer.types";
-import cn from "@/utils/cn";
+import cn from "@/utils/cn/cn";
```

Comment:
like suggestion maybe rename `cn` file to `index` and import just like `import cn from "@/utils/cn"` instead `cn/cn` just for readability
What do you think ?